src/app/forms.py

```diff
@@ -83,6 +84,8 @@ def __init__(self, *args, **kwargs):
     comments = forms.CharField(label='Comments', required=False, widget=forms.Textarea(attrs={'rows': 4, 'cols': 40}))
     licenseHeader = forms.CharField(label='Standard License Header', widget=forms.Textarea(attrs={'rows': 3, 'cols': 40}), required=False)
     text = forms.CharField(label='Text', widget=forms.Textarea(attrs={'rows': 4, 'cols': 40}))
+    repositoryNameWithOwner = settings.DIFF_REPO_WITH_OWNER
+    diffRepoGithubToken = settings.DIFF_REPO_GIT_TOKEN
```

Comment:
Not sure whether passing the secret key in the forms is a good idea. We should have the screenshot function on the backend side for more security, but let's go with this now.